src-tauri/src/main.rs

```diff
@@ -171,7 +171,7 @@ async fn set_mode(
     info!(target: LOG_TARGET, "set_mode called with mode: {:?}, custom_max_cpu_usage: {:?}, custom_max_gpu_usage: {:?}", mode, custom_cpu_usage, custom_gpu_usage);
 
     fn f64_to_isize_safe(_value: Option<f64>) -> Option<isize> {
-        let value = _value.unwrap_or(-1.0);
+        let value = _value.unwrap_or_default();
```

Comment:
`_value` when used should be renamed to `value` as the underscore is used to denote variables that are unused.

Reply:
checking